workers/unity/Assets/Fps/Scripts/UI/Managers/DeploymentListScreenManager.cs

```diff
@@ -45,18 +65,26 @@ public string GetChosenDeployment()
             return null;
         }
 
-        public void SetDeployments(DeploymentData[] deployments)
+        public void SetDeployments(List<DeploymentData> deployments)
         {
+            if (deployments == null)
+            {
+                return;
+            }
+
+            deployments.Sort((deployment1, deployment2) =>
+                String.Compare(deployment1.Name, deployment2.Name, StringComparison.Ordinal));
```

Comment:
You can define an `IComparable<DeploymentData>` on deployment data so you don't need the lambda here.